UPGRADE.md

```diff
@@ -2,6 +2,16 @@
 
 ## dev-develop
 
+### Content Types
+
+The following functions where removed from the `ContentTypeInterface`:
+
+ - getTemplate
+ - getDefaultParams
+ - getDefaultValue
+
+When you have a custom content type you need to register a field type on the same name.
```

Comment:
Somehow explain that this has to happen in JS now?

Reply:
would link here to the documentation ->`Create custom field type` but this is currently not documented.